applications/StructuralMechanicsApplication/tests/test_patch_test_small_strain.py

```diff
@@ -220,7 +220,10 @@ def _check_outputs(self,mp,A,dim):
             for stress in out:
                 for i in range(len(reference_stress)):
                     if abs(stress[i]) > 0.0:
-                        self.assertLess((reference_stress[i] - stress[i])/stress[i], 1.0e-6)
+                        if abs(reference_stress[i]) > 0.0: 
+                            self.assertLess(abs((reference_stress[i] - stress[i])/reference_stress[i]), self.tolerance) 
+                        else: 
```

Comment:
This is not working, as the error as the stresses are huge and the error is bigger than a tolerance defined for a relative error

Reply:
That's not actually the case. ```reference_stress``` is zero everywhere (at least in ```test_SmallDisplacementElement_3D_tetra_user_provided_elasticity```).

Git says @RiccardoRossi added these references. Can you please have a look at them?